.rat-excludes

```diff
@@ -71,6 +71,7 @@ ibm-db2.svg
 postgresql.svg
 snowflake.svg
 ydb.svg
+loading.svg
```

Comment:
```suggestion
```

Just added the license to the SVG file, so you can commit this suggestion to strip out the exclusion :)